Analytics/Config.cs

```diff
@@ -93,5 +95,16 @@ public Config SetAsync(bool async)
 			this.Async = async;
             return this;
         }
+
+		/// <summary>
+		/// Sets the API request header uses GZip option.
```

Comment:
Could we add "why" people should use this. Something like: `Sets the API request header uses GZip option. Enable this when the network is the bottleneck for your application (typically in client side applications).`